src/components/ui/SuggestsDropdown.jsx

```diff
@@ -1,3 +1,4 @@
+/* global _ */
```

Comment:
All changes in this file should be reverted, as the error message condition is controlled from outside now, this dropdown is just rendering stuff without knowing much about them.